VKSample/VKSample/Modules/MyGroupsModule/Cells/MyGroupTableViewCell.swift

```diff
@@ -12,8 +12,23 @@ final class MyGroupTableViewCell: UITableViewCell {
 
     // MARK: - Public methods.
 
-    func configure(with group: Group) {
-        groupImageView.image = UIImage(named: group.titleImageName)
+    func configure(with group: ItemGroup) {
+        guard let groupAvatarText = group.avatar else { return }
+        setImage(userPhotoURLText: groupAvatarText)
         groupTitleLabel.text = group.name
     }
+
+    // MARK: - Private methods.
+
+    private func setImage(userPhotoURLText: String) {
+        let url = URL(string: userPhotoURLText)
+        DispatchQueue.global().async {
+            guard let url = url else { return }
+            let data = try? Data(contentsOf: url)
+            DispatchQueue.main.async {
+                guard let data = data else { return }
+                self.groupImageView.image = UIImage(data: data)
+            }
+        }
+    }
```

Comment:
```suggestion
    private func setImage(userPhotoURLText: String) {
        let url = URL(string: userPhotoURLText)
        DispatchQueue.global().async {
            guard let url = url else { return }
            let data = try? Data(contentsOf: url)
            DispatchQueue.main.async {
                guard let data = data else { return }
                self.groupImageView.image = UIImage(data: data)
            }
        }
    }
```
вынеси в расширение UIimage

Reply:
Плюсую